src/ngraph/op/divide.cpp

```diff
@@ -93,6 +100,13 @@ op::v1::Divide::Divide(const Output<Node>& arg0,
     constructor_validate_and_infer_types();
 }
 
+bool op::v1::Divide::visit_attributes(AttributeVisitor& visitor)
+{
+    BinaryElementwiseArithmetic::visit_attributes(visitor);
+    visitor.on_attribute("m_pythondiv", m_pythondiv);
```

Comment:
What is it `m_pythondiv`?

Also in spec Divide can have only one attribute: `auto_broadcast`.

Reply:
Python and C/C++ do division differently; models expecting the python behavior need the python behavior. For example, https://stackoverflow.com/questions/828092/python-style-integer-division-modulus-in-c